src/rfac/rfmklide.c

```
@@ -0,0 +1,118 @@
+  /* terminate lists */
+  eng[n_eng] = e_int[n_eng] = t_int[n_eng] = (real) F_END_OF_LIST;
+
+  CONTROL_MSG(CONTROL, "(exp) n_eng: %d\n", n_eng);
+ 
+#if WRITE
+  str_e_int = fopen("rfmklist.exp", "w");
+  str_t_int = fopen("rfmklist.the", "w");
+
+  for(i_eng = 0; i_eng < n_eng; i_eng++)
+  {
+    fprintf(str_e_int, "%f %e\n", eng[i_eng], e_int[i_eng]/e_max);
```

Cppcheck (2.19.0)

[warning] If resource allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfResources)